pallets/omnipool/src/lib.rs

```diff
@@ -349,23 +367,19 @@ pub mod pallet {
 		AssetAlreadyAdded,
 		/// Asset is not in omnipool
 		AssetNotFound,
-		/// No stable asset in the pool
-		NoStableAssetInPool,
```

Comment:
This is a breaking change

Reply:
that is just cleanup, it wasn't used anymore

Reply:
Yes, but it changes the index numbers of errors. Maybe it's not a problem.